service/worker/esanalyzer/workflow.go

```diff
@@ -150,18 +149,19 @@ func Workflow(ctx workflow.Context) error {
 	return nil
 }
 
-// RefreshStuckWorkflowsFromSameWorkflowType is activity to refresh stuck workflows from the same domain
-func RefreshStuckWorkflowsFromSameWorkflowType(
+// refreshStuckWorkflowsFromSameWorkflowType is activity to refresh stuck workflows from the same domain
+func refreshStuckWorkflowsFromSameWorkflowType(
 	ctx context.Context,
 	workflows []WorkflowInfo,
 ) error {
+	logger := activity.GetLogger(ctx)
 	analyzer := ctx.Value(analyzerContextKey).(*Analyzer)
 	domainID := workflows[0].DomainID
 	domainEntry, err := analyzer.domainCache.GetDomainByID(domainID)
 	if err != nil {
-		analyzer.logger.Error("Failed to get domain entry",
-			tag.WorkflowDomainID(domainID),
-		)
+		logger.Error("Failed to get domain entry",
+			zap.String("error", fmt.Sprintf("%v", err)),
```

Comment:
there is a `zap.Error` for these purposes fwiw.

more generally, eventually we should convert these to something like `return errors.WrapWith("Failed to get domain entry", err, zap.String(...))` rather than double-logging (once here with useful info, once with less info when the activity-logger logs the error).